api/src/paths/public/plans.ts

```diff
@@ -354,6 +355,15 @@ export function getPublicPlansList(): RequestHandler {
       // Get all projects data for the projectIds
       const projects = await planService.getPlansByIds(projectIds, true);
 
+      // Mask geometries that require it
+      projects.forEach((project) => {
```

Comment:
You broke the tests for each of these endpoints, could you update them accordingly?

Reply:
Fixed now.